starlite_oidc/middleware.py

```diff
@@ -1,4 +1,4 @@
-from typing import Literal, Optional
+from typing import Literal
```

Comment:
Use `from typing_extensions import Literal` here. It is not supported in earlier python versions